Source/ChocolateyGui.Common.Windows/Services/DialogService.cs

```diff
@@ -200,5 +201,15 @@ public async Task<LoginDialogData> ShowLoginAsync(string title, string message,
                 return default;
             }
         }
+
+        private static string L(string key)
+        {
+            return TranslationSource.Instance[key];
+        }
+
+        private static string L(string key, params object[] parameters)
+        {
+            return TranslationSource.Instance[key, parameters];
+        }

```

Comment:
This is repeated a few times for services... Is it worth creating a BaseService class that has these methods?  Similar to what was done with the new ViewModelScreen class?

Reply:
I was honestly thinking about it but didn't want to pollute a base class with just static methods.

I could still do it though, but it wouldn't be the same as the ViewModelScreen base class as that also support passing in a distinct TranslationSource which wouldn't be possible for the classes in the service that require a static method.

Reply:
If they're all static method, we could stick them in a shared helper class, it needn't (necessarily) be a base class.

Reply:
@vexx32 technically the helper class is the `TranslationSource` class, the methods here are just for convenience.

Reply:
@AdmiringWorm gotcha. Makes sense. Based on what you have said here, I am happy to leave this the way it is.